BootloaderCommonPkg/Include/Library/FirmwareUpdateLib.h

```diff
@@ -61,8 +70,8 @@ typedef enum {
 } BOOT_PARTITION;
 
 #define ESRT_FIRMWARE_RESOURCE_VERSION    0x1
-#define ESRT_FIRMWARE_RESOURCE_COUNT      0x1
-#define ESRT_FIRMWARE_RESOURCE_COUNT_MAX  0x1
+#define ESRT_FIRMWARE_RESOURCE_COUNT      0x3
```

Comment:
why define ESRT_FIRMWARE_RESOURCE_COUNT, is there anyone using it?

Reply:
Removed them, Using MAX_FW_COMPONENTS instead